src/app/projects/projects.state.ts

```diff
@@ -235,6 +235,11 @@ export class ProjectsState {
         pyYw.Routers.Projects.Project[]
     >([])
 
+    /**
+     * @group Observables
+     */
+    public readonly dropdownHandler$ = new BehaviorSubject<boolean>(false)
```

Comment:
This observable is very specific to  `SectionUnloadedProjects` and also very specific to a detail implementation of the view (it is not meant to really be exposed at other place).

I would rather have this attribute declare in `SectionUnloadedProjects` .